### DISCUSSION TITLE
Export custom selections of waypoints to any format

### DISCUSSION THREAD

**Discussion by Tazzzilo** [albar965/littlenavmap#1020] [Ideas]
I noticed that when you select multiple Waypoints/Airports in your flight plan that you are only able to export them via right click context-menu to .lnmpln format.

You can't choose the format like you are able to do via main menu bar (File -> Export...)

It would be nice to select a number of custom waypoints and export them to every custom format. 

**Why is this helpful?** 
If you log every destination in your current flight plan and add the new destinations at the end that you are still able to just create a flight plan file for the new destinations.

**Comment by albar965** [comment]
Export selected flight plan legs for all formats? Not sure. You can still modify the plan and then export.

**Comment by Tazzzilo** [comment]
It would be sufficient from my point of view if you select the legs you want export and do the normal exporting via menu bar and the export dialog asks you if you want to export the whole flight plan or only the legs you have selected. Only exporting a certain leg from your whole flightplan for example to the Garmin GTN750 is better to handle in game. 
I mean a selection like this: 
![image](https://user-images.githubusercontent.com/31939403/235358118-347f8a6f-3e7b-4362-8063-1377d43be4ca.png)

Background: 
When using FSE Economy you often plan a lot of legs for today flight and fly them one by one or even the next day. This will make using the Garmin easier when you only have exported one leg instead of all

**Reply by albar965** [reply]
Ok. Maybe something like `Flight Plan` -> `Multiexport selected` .

**Reply by albar965** [reply]
Created issue #1021
